### ISSUE TITLE
Issues Running anaconda-projects commands in a subdirectory

### ISSUE THREAD

**Issue by AlbertDeFusco** [Anaconda-Platform/anaconda-project#190]
When I run `anaconda-project` commands in a subdirectory of my project I end up with new `anaconda-project.yml` files in those subdirectories.

## 1. Prepare project
Here's my `anaconda-project.yml` file

```yaml
name: subdirectories
  
packages:
    - python=3.7
    - requests
channels: []

env_specs:
  env:
    packages: []
    channels: []
```

then I run `anaconda-project prepare`

## 2. Make subdirectory
```
> mkdir dir1
```

## 3. Attempt to add a package from within subdirectory
```
> cd dir1
> anaconda-project add-packages numba
```

I end up with a new `anaconda-project.yml` in `dir1` with the contents consistent with running the following two commands in `dir1`.

```
> anaconda-project init
> anaconda-project add-packages numba
```

**Comment by mcg1969** [comment]
Albert, do you think we should be behaving just like Git here? That is to say, no matter how many subdirectories up an `anaconda-project.yml` file is found, it should be assumed that represents the root of the project?

**Comment by AlbertDeFusco** [comment]
Yes, I do like that approach.

**Comment by mcg1969** [comment]
Ok, I agree. The challenge is implementing this... I will take a look but other eyeballs welcome

**Comment by AlbertDeFusco** [comment]
I appear to have it working at this branch https://github.com/AlbertDeFusco/anaconda-project/tree/subdirectories

I'll work on a few tests

**Comment by mcg1969** [comment]
One thing we want to make sure of is that an explicit call to `anaconda-project init` does ignore subdirectories and places its results in the current directory≥

**Comment by AlbertDeFusco** [comment]
So in this case you'd prefer that nested anaconda-project.yml files be created when `init` is called?

```
project/
  - anaconda-project.yml
  - subdir1/
    - file
```

In this case the following command should create a new `anaconda-project.yml` in `project/subdir1`?

```
cd project/subdir1
anaconda-project init
```

**Comment by mcg1969** [comment]
I could see a warning being raised here, but if someone is doing `anaconda-project init` they are presuming that the file does not yet exist. It seems unlikely that they would not already know the file exists. So yes, I think it should be placed right in the current directory, regardless. But if `git init` does something different I can also be persuaded—perhaps a warning or even an error be raised instead?

**Comment by AlbertDeFusco** [comment]
Got it working. Like git no warning is presented if init is run from within a project subdirectory. I'll do some more testing and verification before submitting a PR.

**Comment by AlbertDeFusco** [comment]
This is fixed by PR #215
